### PR TITLE
Qualifications

### PR DESCRIPTION
**Pull request by ChazUK**
Add / View / Edit / Delete Qualifications

I also updated the HTTP Error Handler to throw an error on Unauthorised as it was chucking out a load of console logs.

### CODE REVIEW

**Comment by ChazUK** [comment]
[Add/ Update worker qualifications #88](https://trello.com/c/zkIxMNm0/340-add-update-worker-qualifications-88)

File: src/app/core/services/worker.service.ts [hunk 1]
```diff
@@ -111,6 +121,57 @@ export class WorkerService {
     );
   }
 
+  getAvailableQualifcations(workerId: string, type: QualificationType) {
```
Comment (wozzer72) [inline src/app/core/services/worker.service.ts]: I see you've put all the qualifications and training APIs in the same worker service. A bit late to change now, but separate qualifications and separate training services would have been more encapsulated.

Comment (ChazUK) [reply src/app/core/services/worker.service.ts]: I could pull them about but I still see these calls as directly Worker related. Happy to move it if you like.

File: src/assets/scss/forms.scss [hunk 2]
```diff
@@ -91,6 +91,10 @@ form,
     .panel-border-narrow {
 
       @include rem(margin, 20px 0 0 20px);
+
+      &.no-pad {
```
Comment (denny13) [inline src/assets/scss/forms.scss]: no-pad - referencing margin?

You can use a class of 'ml-0' to achieve the same thing also

**Pull request merged**